datalad/customremotes/base.py

```diff
@@ -321,7 +321,7 @@ def _loop(self):
                 method(*req_load)
             except Exception as e:
                 self.error("Problem processing %r with parameters %r: %r"
-                           % (req, req_load, exc_str(e)))
+                           % (req, req_load, str(exc_str(e))))
```

Comment:
What makes this call to `exc_str()` (together the few additional ones below) different from the ~150 other calls that are not adjusted?

Reply:
why `str()` here -- if needed, may be just replace last `%r` with `%s`?
or overall -- why to not chase all `exc_str` invocations to be placed under `str()` and just make the new `exc_str` to return that string representation and have e.g.

```
def exc_cap(e: Exception) -> CapturedException:
   return CapturedException.capture_exc(e)
```
to use where non-str form is desired.  This would minimize the diff I guess making it easier to grasp etc.

Reply:
> may be just replace last %r with %s

Yeah, good point.